fixtures/tracks/fruit/SETUP.md

```diff
@@ -1 +1 @@
-The SETUP.md file is deprecated, and exercises/TRACK_HINTS.md should be used.
+The SETUP.md file is deprecated, and docs/EXERCISE_README_INSERT.md should be used.
```

Comment:
Want to update the fixtures for TRACK_HINTS.md as well to say that they're deprecated?